tests/test_isrTaskLSST.py

```diff
@@ -350,8 +354,9 @@ def test_isrDark(self):
         isr_config = self.get_isr_config_electronic_corrections()
         isr_config.doBias = True
         isr_config.doDark = True
-        # We do not do defect correction when processing darks.
+        # We do not do these corrections when processing darks.
         isr_config.doDefect = False
+        isr_config.doDeferredCharge = False
```

Comment:
Similarly, we will be doing deferred charge correction when processing regular darks.